membership/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.conf import settings
+# Create your models here.
+
+MEMBERSHIP_CHOICES = (
+('Premium', 'pre'),
+('Free', 'free')
+)
+class Membership(models.Model):
+    slug = models.SlugField(null=True, blank=True)
+    membership_type = models.CharField(
+    choices=MEMBERSHIP_CHOICES, default='Free',
+    max_length=30
+      )
```

Comment:
Let's work on writing nicer code. 

```python
    membership_type = models.CharField(
        choices=MEMBERSHIP_CHOICES, default='Free',
        max_length=30
    )
````
`default='free'` <- you'll need that when you update the tuple. 
`max_length=7` <- max length only needs to be as long as your maximum value (7 characters for the word "premium")